apps/weather_app/src/app/core/models/cities.mode.ts

```diff
@@ -3,9 +3,23 @@ export type SimpleCityType = {
   name: string;
 };
 
-export type GeoCityData = {
+export interface GeoCityData {
   name: string;
   latitude: number;
   longitude: number;
   country: string;
+  state?: string;
+}
+
+export type CityWeather = {
```

Comment:
- should it be type, can it be interface?
- it is better to keep common style (camel case as recommended for all types/objects/enums, snake case is used here)